sections/image-banner.liquid

```diff
@@ -425,6 +470,12 @@
           ],
           "default": "body",
           "label": "t:sections.image-banner.blocks.text.settings.text_style.label"
+        },
+        {
+          "type": "text",
+          "id": "bold_text",
+          "label": "Bold Text",
+          "info": "Enter the bold text to highlight."
```

Comment:
why create input field for bold text when you can bold it in other fields as well?